src/main/java/org/lightcouch/View.java

```diff
@@ -381,7 +383,7 @@ public boolean queryForBoolean() {
                 boolean isLastPage = resultRows <= rowsPerPage;
                 if (!isLastPage) {
                     page.setHasNext(true);
-                    jsonNext.addProperty(START_KEY, rows.get(i).getKey());
+                    jsonNext.addProperty(START_KEY, rows.get(i).getKey().toString());
```

Comment:
In order to fix the original problem with the queryView causing an exception, I changed the first parameter of the queryView constructor from String.class to Object.class (line 365).  Now, the `getKey()` method above will return an object.  I needed to call `toString` to get the string representation and set that to the start key.  

In my latest tests, I'm finding that key arrays with one string value are missing the quotes by the time it reaches this code.  For example, a key with "["abc"]" or "["1"]" would show as "[abc]" or "[1]".  Here is a gist of an if block that would add the required quotes:
https://gist.github.com/emlaver/81cc2db5cbfc0291ff8e

I'd like to replace the code in the gist with something more efficient.  I may need to manipulate the key in the `JsonToObject` method of CouchDBUtil.java.